tensorflow_probability/python/distributions/laplace.py

```diff
@@ -201,3 +202,26 @@ def _mode(self):
 
   def _z(self, x):
     return (x - self.loc) / self.scale
+
+@kullback_leibler.RegisterKL(Laplace, tf.distributions.Laplace)
+@kullback_leibler.RegisterKL(tf.distributions.Laplace, Laplace)
+@kullback_leibler.RegisterKL(Laplace, Laplace)
+def _kl_laplace_laplace(a, b, name=None):
+    """Calculate the batched KL divergence KL(a || b) with a and b Laplace.
+
+    Args:
+      a: instance of a Laplace distribution object.
+      b: instance of a Laplace distribution object.
+      name: (optional) Name to use for created operations.
+        default is "kl_laplace_laplace".
+
+    Returns:
+      Batchwise KL(a || b)
+    """
+    with tf.name_scope(name, 'kl_laplace_laplace', [a.loc, b.loc, a.scale, b.scale]):
+        # Consistent with http://www.mast.queensu.ca/~communications/Papers/gil-msc11.pdf, page 38
```

Comment:
just eyballing, this looks possibly >80 chars. can you follow the [instructions from TF](https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#python-coding-style) on running a linter on the code?